x/staking/keeper/alias_functions.go

```diff
@@ -75,26 +93,17 @@ func (k Keeper) IterateLastValidators(ctx sdk.Context, fn func(index int64, vali
 		}
 		i++
 	}
+	return nil
 }
 
 // Validator gets the Validator interface for a particular address
-func (k Keeper) Validator(ctx sdk.Context, address sdk.ValAddress) types.ValidatorI {
-	val, found := k.GetValidator(ctx, address)
-	if !found {
-		return nil
-	}
-
-	return val
+func (k Keeper) Validator(ctx context.Context, address sdk.ValAddress) (types.ValidatorI, error) {
+	return k.GetValidator(ctx, address)
```

Comment:
This is a behavior change, before if the validator didn't exist it returned nil, but now it returns `types.ErrNoValidatorFound`.
How this alias should still behave?

Reply:
I believe a good option would be returning `nil, nil` in case of "not found"

Reply:
Idiomatic Go, with a `<*, error>` return type, would indicate that a non-nil error should be returned in case the record/object does not exist, even better to return a sentinel/typed error, e.g. `ErrRecordNotFound`.